project-management/src/main/java/life/qbic/projectmanagement/application/dataset/RawDataService.java

```diff
@@ -114,7 +115,8 @@ public Collection<RawData> findNGSRawData(String filter,
                       .equals(datasetInformation.measurementCode()))
               .findFirst().orElseThrow();
           var sampleInformation = sampleInformationService.retrieveSamplesByIds(
-                  measurement.measuredSamples()).stream()
+                  measurement.specificMeasurementMetadata().stream().map(
+                      NGSSpecificMeasurementMetadata::measuredSample).toList()).stream()
```

Comment:
```suggestion
                  measurement.specificMeasurementMetadata().stream()
                  .map(NGSSpecificMeasurementMetadata::measuredSample)
                  .toList())
                  .stream()
```
When you put the consecutive stream calls in a new line each it is easy to see that here, a stream is collected into a list only to be streamed directly after again.
You can extract a variable here to store the list with a meaningful name or you can remove the `.toList().stream()` chaining.